x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -413,9 +460,19 @@ export const InnerWorkspacePanel = React.memo(function InnerWorkspacePanel({
           )
         : undefined;
 
-    return customWorkspaceRenderer ? (
-      customWorkspaceRenderer()
-    ) : (
+    if (customWorkspaceRenderer) {
+      return customWorkspaceRenderer();
+    }
+
+    const hasSomethingToRender = localState.expressionToRender !== null;
+
+    const renderWorkspaceContents = hasSomethingToRender
+      ? renderVisualization
+      : !changesApplied && !hasSomethingToRender
+      ? renderApplyChangesPrompt
+      : renderDragDropPrompt;
```

Comment:
wouldn't this be the same? 

```suggestion
    const renderWorkspaceContents = hasSomethingToRender
      ? renderVisualization
      : !changesApplied
      ? renderApplyChangesPrompt
      : renderDragDropPrompt;
```